### PR TITLE
Aguinaga_InputTTL

### PR DESCRIPTION
**Pull request by ponchies12**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Kate:
 
 
 Jon:
-1. Your response here.
-2. And here...
-3. Also here!
+1. I am allergic to melon.
+2. I in-line skate.
+3. I accidentally put in a cup of noodles in the microwave with no water...
```
Comment (1010sheehy) [inline README.md:17]: I might have done this too...

File: README.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ Kate:
 
 
 Jon:
-1. Your response here.
-2. And here...
-3. Also here!
+1. I am allergic to melon.
```
Comment (kllaskowski) [inline README.md:15]: Is this the reason why you really love berries?

File: README.md [hunk 3]
```diff
@@ -12,9 +12,9 @@ Kate:
...
+2. I in-line skate.
```
Comment (kllaskowski) [inline README.md:16]: You've never mentioned this....

File: README.md (hunk 1 above)
Comment (kllaskowski) [inline README.md:17]: I have probably done this at some point...

Comment (caiquelegreat) [reply README.md:17]: In-line skating?

**Pull request merged**